plugins/UM3NetworkPrinting/src/Network/SendMaterialJob.py

```diff
@@ -127,20 +128,18 @@ def _sendingFinished(self, reply: QNetworkReply) -> None:
     @staticmethod
     def _getLocalMaterials() -> Dict[str, LocalMaterial]:
         result = {}  # type: Dict[str, LocalMaterial]
-        material_manager = CuraApplication.getInstance().getMaterialManager()
-        material_group_dict = material_manager.getAllMaterialGroups()
+        all_materials = CuraApplication.getInstance().getContainerRegistry().findInstanceContainersMetadata(type = "material")
+        all_root_materials = [material for material in all_materials if material["id"] == material.get("base_file")]  # Don't send materials without base_file: The empty material doesn't need to be sent.
```

Comment:
Same as above, I'd name it `all_root_material_ids`.

Reply:
Done: 05de920c21cc3cf262ecb376de86e6944c3852bb